### DISCUSSION TITLE
[x]it vim support

### DISCUSSION THREAD

**Discussion by sadotsoy** [jotaen/xit#36] [Show & tell]
Hi, I got inspired by [@ryanolsonx plugin](https://github.com/ryanolsonx/vim-xit) to expand it with my personal salt taste.

[sadotsoy/vim-xit](https://github.com/sadotsoy/vim-xit)
 
I've added a todo list for the next features if you want to participate.

current status v0.1:
![darkforce](https://user-images.githubusercontent.com/11858725/164130278-bd1630b7-63c5-4890-9f94-dd243ca64563.png)

**Comment by sadotsoy** [comment]
Don't hate me I like the empty checkbox `[]` So is valid for default, I've added a functionality if you want to invalide it [see more](https://github.com/sadotsoy/xit-vim#notes)

**Reply by jotaen** [reply]
Awesome that you started this! I think there is enough room for multiple plugins for the same editor.

I’m happy to link your project from the website, let me know if or when I should do this.

> Don't hate me I like the empty checkbox `[]` So is valid for default, I've added a functionality if you want to invalide it [see more](https://github.com/sadotsoy/xit-vim#notes)

I think it’s cool that you made your `[]` variant explicit via a configuration, that way it should be clear that it’s a custom “flavour”.

One note: the [spec defines indentation to be precisely 4 spaces](https://github.com/jotaen/xit/blob/main/Specification.md#description), so when using a shorter checkbox sequence, the description lines [don’t align nicely anymore](https://github.com/sadotsoy/xit-vim/blob/d8c4ac9a942d07a5300709dabc25157d4d0993f2/todo.xit#L7-L9). The 4 spaces for indentation were carefully chosen so that the text of all description lines start at the same column (namely, the fifth one).

```
[ ] This is a description.
    The next line starts at the fifth column.
    That way, all lines are aligned with each other.
```

**Reply by sadotsoy** [reply]
Hi @jotaen thanx for the note, I'm going to work the indentation as a feature for the next version, for now you can link the project into your web actually I've move the repo to this one https://github.com/sadotsoy/vim-xit I've just renamed the repo from `xit-vim` to `vim-xit`

**Reply by jotaen** [reply]
I’ve added the link on the website.
